src/components/Calendar/util.ts

```diff
@@ -98,3 +98,14 @@ export const isValidDate = (possibleDate: any): boolean => {
   // eslint-disable-next-line no-self-compare
   return possibleDate && possibleDate.getTime() === possibleDate.getTime()
 }
+
+/**
+ * Check if the parameter is a valid ReferenceMonth.
+ * @param possibleReferenceMonth - The ReferenceMonth object to be validated.
+ * @returns True if the ReferenceMonth is valid, false otherwise.
+ */
+export const isValidReferenceMonth = (possibleReferenceMonth: any): boolean => {
```

Comment:
Why don't you specify the type here as `ReferenceMonth`? 

The documentation and the function name imply that the variable should have this type.

Reply:
I took the `isValidDate` function as an example, but actually, I think that makes more sense to specify the type here